### DISCUSSION TITLE
Justify text in page layout?

### DISCUSSION THREAD

**Discussion by DMatsliyah** [alshedivat/al-folio#1965] [Q&A] [answered]
Hi all
I've been trying to figure out how to justify paragraphs written in 'page' layout, I'm working on pages specifically assigned to projects.

I tried adding `<p class="text-justify"> </p>` that I've seen in a [bootstrap doc](https://getbootstrap.com/docs/4.0/utilities/text/) but I think this may either be a wrong version or irrelevant altogether?

Is there an easy fix to this? I have zero experience with this type of coding and have been done a ton of trial and error throughout setting up this template. (Sweet project by the way!)

**Comment by george-gca** [comment, marked as answer]
You can add
```css
p {
  text-align: justify;
}
```
in [_sass/_base.scss](https://github.com/alshedivat/al-folio/blob/master/_sass/_base.scss).
